symfony-bundle/key-and-key-set-management/key-set-management-jwkset.md

```diff
@@ -87,15 +87,17 @@ jose:
                 url: 'https://www.googleapis.com/oauth2/v1/certs'
 ```
 
-## Shared Ket Sets
+## Shared Key Sets
 
 It can be interesting to share your key sets through an Url. This can easily achieved by adding a dedicated controller. This controller is automatically created by the bundle.
 
-You must add the following configuration to your routing file.
+You can enable these routes by adding the following configuration to your routing file.
 
 ```yaml
+# config/routes.yaml
 jwkset_endpoints:
-    resource: "@JoseFrameworkBundle/Resources/config/routing/jwkset_controller.yml"
```

Comment:
Indeed the correct path is @JoseFrameworkBundle/Resources/config/routing/jwkset_controller.**php**, not **yml**
I think it is preferrable to keep it. WDYT?

Reply:
You're totally right, I missed that file for some reason, but found the routing loader class :)
Now I modified it to work with the **php** resource version.